apps/ex_wire/test/ex_wire/kademlia/routing_table_test.exs

```diff
@@ -60,7 +54,7 @@ defmodule ExWire.Kademlia.RoutingTableTest do
       assert Enum.count(pongs) == 1
 
       {_mdc, pair} = Enum.at(pongs, 0)
-      assert pair == {filler_node, node}
+      {^filler_node, ^node, _} = pair
```

Comment:
Since this is a test, should we be `assert`ing something?

Reply:
@germsvel  we pattern match here. if the right part does not have `filler_node`, `node` as first tuple elements, it will fail. but I can change it to assert

Reply:
Yeah, I know the pattern match will fail the test if it fails. But I tend to like the assertion because it makes it more clear what we are testing. Without the `assert`, on first glance, it seems like we forgot to finish the test. What do you think?